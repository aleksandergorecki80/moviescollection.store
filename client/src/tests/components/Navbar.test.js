// react-test-renderer
import React from 'react';
import { shallow } from 'enzyme';
// import toJSON from 'enzyme-to-json';
import Navbar from '../../components/Navbar';

test('Should render Navbar', () => {
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Navbar />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();


    // expect(wrapper.find('h1').length).toBe(1);
    // expect(wrapper.find('h1').text()).toBe("The collection of my films.");
});