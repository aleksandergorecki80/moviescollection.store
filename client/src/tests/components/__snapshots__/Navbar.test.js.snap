// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Navbar 1`] = `
<div
  className="navbar"
>
  <h1>
    The collection of my films.
  </h1>
  <[object Object]
    to="/"
  >
    Home
  </[object Object]>
  <[object Object]
    to="/add_film"
  >
    Add new film
  </[object Object]>
</div>
`;
